foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -792,23 +794,23 @@ msgstr ""
 "Der Router kann eingestellt werden, um Ihnen Nachrichten über bestimmte "
 "Ereignisse zu senden; zum Beispiel, dass ein Neustart erforderlich ist, dass "
 "kein Speicherplatz mehr vorhanden ist, oder dass eine "
-"Anwendungsaktualisierung installiert wurde. Ihr könnt die Turris-Server für "
-"das Absenden dieser Nachrichten verwenden. Alternativ könnt Ihr auch eigene "
+"Anwendungsaktualisierung installiert wurde. Sie können die Turris-Server für "
+"das Absenden dieser Nachrichten verwenden; Alternativ können Sie auch eigene "
 "SMTP-Server dafür benutzen, wofür aber weitere Einstellungen benötigt "
-"werden. Es sind meistens die selben Einstellungen, die Ihr schon in Ihrem E-"
-"Mail-Client verwendet. Ihr könnt diese Einstellungen von Ihrem E-Mail-"
-"Serviceprovider erhalten. In diesem Fall raten wir Ihnen, aus "
-"Sicherheitsgründen ein neues Benutzerkonto für den Router einzurichten."
+"werden. Es sind meistens die selben Einstellungen, welche Sie schon in Ihrem "
+"E-Mail-Client verwenden. Diese können Sie von Ihrem E-Mail-Serviceprovider "
+"erhalten. In diesem Fall raten wir Ihnen, aus Sicherheitsgründen ein neues "
+"Benutzerkonto für den Router einzurichten."
 
 msgid ""
 "Also, when an automatic restart is required, you can specify the time you "
 "want it to occur. If you have email notifications enabled, you can also "
 "specify the interval between notification and automatic restart."
 msgstr ""
-"Falls ein Neustart benötigt wird, könnt Ihr die genaue Uhrzeit des Neustarts "
-"bestimmen. Wenn Ihr das Absenden der Meldungen über Ereignisse aktiviert "
-"habt, könnt Ihr auch die Zeitspanne zwischen dem Absenden und dem Neustart "
-"bestimmen."
+"Falls ein Neustart benötigt wird, können Sie die genaue Uhrzeit des "
+"Neustarts bestimmen. Wenn Sie die Benachrichtigung per eMail über Ereignisse "
```

Comment:
I would write 'E-Mail' to keep the term consistent throughout the entire German text.